internal/kafka/internal/config/kafka_supported_instance_types.go

```diff
@@ -49,16 +49,21 @@ func (kp *KafkaInstanceType) validate() error {
 
 type KafkaInstanceSize struct {
 	Id                          string   `yaml:"id"`
+	DisplayName                 string   `yaml:"display_name"`
 	IngressThroughputPerSec     Quantity `yaml:"ingressThroughputPerSec"`
 	EgressThroughputPerSec      Quantity `yaml:"egressThroughputPerSec"`
 	TotalMaxConnections         int      `yaml:"totalMaxConnections"`
 	MaxDataRetentionSize        Quantity `yaml:"maxDataRetentionSize"`
 	MaxPartitions               int      `yaml:"maxPartitions"`
 	MaxDataRetentionPeriod      string   `yaml:"maxDataRetentionPeriod"`
 	MaxConnectionAttemptsPerSec int      `yaml:"maxConnectionAttemptsPerSec"`
+	MaxMessageSize              Quantity `yaml:"maxMessageSize"`
 	QuotaConsumed               int      `yaml:"quotaConsumed"`
 	QuotaType                   string   `yaml:"quotaType"`
 	CapacityConsumed            int      `yaml:"capacityConsumed"`
+	SupportedAZModes            []string `yaml:"supportedAZModes"`
+	MinInSyncReplicas           int      `yaml:"minInSyncReplicas"` // also abbreviated as ISR in Kafka terminology
+	ReplicationFactor           int      `yaml:"replicationFactor"` // also abbreviated as RF in Kafka terminology
 }
 
 // validates Kafka instance size configuration to ensure the following:
```

Comment:
Can we add some validation for the new fields? 

- `DisplayName` shouldn't be an empty string.
- `MaxMessageSize` should be a valid K8s quantity. 
- `SupportedAZModes` should have at least 1 length, optionally we could even validate that the list only includes valid input ("multi" or "single"). 
- `MinInSyncReplicas` and `ReplicationFactor` should be greater than 0.

Reply:
ah just saw your comment there about verification :) Can ignore this!

The new field names/format looks good to me. Will wait on @wtrocki to confirm.

Reply:
Added validations